components/rmrk/Gallery/History.vue

```diff
@@ -123,19 +125,37 @@ import shortAddress from '@/utils/shortAddress'
 import { formatDistanceToNow } from 'date-fns'
 import { exist } from '@/components/rmrk/Gallery/Search/exist'
 import { Debounce } from 'vue-debounce-decorator'
-
 const components = {
   Identity: () => import('@/components/shared/format/Identity.vue'),
   Pagination: () => import('@/components/rmrk/Gallery/Pagination.vue'),
 }
 
+const enum HistoryEventType {
```

Comment:
Why don't we use types from `minimark` ?

Reply:
Because the `interaction` from `minimark ` lack `UNLIST` event.

And also there some events are unsuitable for the history component to display.

<img width="327" alt="image" src="https://user-images.githubusercontent.com/31397967/168613479-c86f8c23-ffbb-4746-b63a-32dfaba7b2c4.png">